lib/processor.js

```diff
@@ -170,23 +166,26 @@ function parse(text) {
 
         // Line content
         } else {
-            next = match(/[^\r\n]*/);
+            next = match(/^[^\r\n]*/);
```

Comment:
Will this only grab one line now? Before it was returning everything to end of the file.

If it is different from the `next` as used above, maybe it deserves a different name? `nextLine` maybe?